src/arch/arm/config.cmake

```diff
@@ -208,6 +208,17 @@ config_option(
 )
 mark_as_advanced(KernelAArch64SErrorIgnore)
 
+config_option(
+    KernelAllowSMCCalls ALLOW_SMC_CALLS "Allow components to make SMC calls.
+    WARNING: Some platforms require SMC calls to actually be made to boot VMs \
+    correctly, however allowing SMC calls causes a couple of issues. Since seL4 \
+    cannot pre-empt the supervisor the WCET is no longer guaranteed. Also, \
```

Comment:
```suggestion
    cannot pre-empt the supervisor, the WCET is no longer guaranteed. Also, \
```

Reply:
Also, I think we should use "monitor" (or "secure monitor") instead of "supervisor" here.

Reply:
Updated both suggestions.